compiler/src/dmd/todt.d

```diff
@@ -1437,6 +1437,13 @@ private extern (C++) class TypeInfoDtVisitor : Visitor
                 /* ti.toObjFile() won't get called. So, store these
                  * member functions into object file in here.
                  */
+
+                if (sd.semanticRun < PASS.semantic3done)
+                {
+                    import dmd.semantic3 : semanticTypeInfoMembers;
+                    semanticTypeInfoMembers(sd);
```

Comment:
Is this a sensible place to run sema

Reply:
We are about to emit the functions here, so we need sema3.